girder/models/user.py

```diff
@@ -46,6 +47,11 @@ def initialize(self):
         self.exposeFields(level=AccessType.ADMIN, fields=(
             'size', 'email', 'groups', 'groupInvites'))
 
+        events.bind('model.user.save.created', 'grantSelfAccess',
+                    self._grantSelfAccess)
+        events.bind('model.user.save.created', 'addDefaultFolders',
+                    self._addDefaultFolders)
```

Comment:
> an enum in `girder.constants`

Great idea. May be this could be done in a follow up PR and applies to the entire code base ?